packages/fusion-runtime/src/unifiedGraphManager.ts

```diff
@@ -426,7 +430,7 @@ export class UnifiedGraphManager<TContext> implements AsyncDisposable {
       );
     } catch (e) {
       this.opts.transportContext?.logger?.error('Failed to load Supergraph', e);
-      this.continuePolling();
+      this.lastLoadTime = Date.now();
```

Comment:
Maybe we might not set the load time here, having the next request try pulling the schema again?

Reply:
I considered polling here as an attempt and waiting for some time will give the source some backoff.